CHANGELOG.md

```diff
@@ -13,6 +13,11 @@
 
 ## Master
 
+# 3.8.3
```

Comment:
Adding this means likely merge conflicts, best to leave them next time - I normally add this version when I create the new release. I should make a danger rule for that 👍

Reply:
I can avoid adding changelog for the next PR. Do you prefer that?